Automattic/wp-calypso
# Signup reskin: Domain step header changes

#### Changes proposed in this Pull Request

This PR is towards implementing the reskinned signup experiment. For more context on the experiment, read pbxNRc-IO-p2.

NOTE:
This PR implements only the header section of the domain step. The other changes will be implemented in subsequent PRs.

Here's the [Figma file](https://www.figma.com/file/gnYBUOBthTMckJRJr3pZmk/Control-Experiments?node-id=1114%3A536) for reference.

Parent PR: https://github.com/Automattic/wp-calypso/pull/50512

<img width="1668" alt="Screenshot 2021-03-31 at 3 51 51 PM" src="https://user-images.githubusercontent.com/1269602/113140181-aeeb7f80-9245-11eb-96c0-4fc5f875b7be.png">

#### Testing instructions



* Assign yourself to treatment from Abacus, and visit /start.
* Proceed to the domains step and verify that the domain step header matches the [Figma design](https://www.figma.com/file/gnYBUOBthTMckJRJr3pZmk/Control-Experiments?node-id=1114%3A536).
* Reload the page on the domains step and verify that the reskinned signup UI is displayed.
* Click on "Use a domain I own" and verify that there's no header for the mapping/transfer section.
* Start over and assign yourself to control this time from Abacus. Visit /start and verify that control experience is unchanged.
